<launch>
    <node name="camera" pkg="usb_cam" type="usb_cam_node" output="screen" >
        <param name="video_device" value="/dev/video1" />
        <param name="image_width" value="1280" />
        <param name="image_height" value="720" />
        <param name="pixel_format" value="mjpeg" />
        <param name="camera_frame_id" value="webcam" />
        <param name="camera_info_url" type="string" value="file://$(find usb_cam)/cal.yml" />
    </node>
  <group ns="camera">
    <node pkg="image_proc" type="image_proc" name="image_proc_node" output="screen">
    </node>
  </group>

  <node pkg="line_detection" name="backproject-skeletonize" type="backproject-skeletonize.py" output="screen">
  </node>

  <node pkg="image_view" name="image_view_rect" type="image_view" output="screen">
    <remap from="image" to="/camera/image_rect_color" />
  </node>

  <node pkg="image_view" name="image_view_raw" type="image_view" output="screen">
    <remap from="image" to="/camera/image_raw" />
  </node>

  <node pkg="image_view" name="image_view_line" type="image_view" output="screen">
    <remap from="image" to="/line_image" />
  </node>
</launch>